qiskit_experiments/database_service/db_experiment_data.py

```diff
@@ -1381,27 +1399,6 @@ def __repr__(self):
         out += ")"
         return out
 
-    def __str__(self):
```

Comment:
I don't think the base class should have a pretty print repr since we don't want users to directly use it. This just makes it easier to confuse the DbExperimentData with the intended user facing ExperimentData